graphs/adapter-api/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/api/GraphCheckFactory.kt

```diff
@@ -4,7 +4,9 @@ import com.anaplan.engineering.azuki.core.system.Check
 import com.anaplan.engineering.azuki.core.system.CheckFactory
 import com.anaplan.engineering.azuki.core.system.UnsupportedCheck
 
-interface GraphCheckFactory: CheckFactory {
+interface GraphCheckFactory : CheckFactory {
     fun hasVertexCount(graphName: String, count: Long): Check = UnsupportedCheck
     fun hasShortestPath(graphName: String, from: Any, to: Any, shortestPath: List<Any>): Check = UnsupportedCheck
+    fun hasCycle(graphName: String, result: Boolean): Check = UnsupportedCheck
+    fun hasCycleCount(graphName: String, result: Long): Check = UnsupportedCheck
```

Comment:
⛏️ For consistency with the pre-existing methods, I would prefer more informative parameter names:
```suggestion
    fun hasCycle(graphName: String, hasCycle: Boolean): Check = UnsupportedCheck
    fun hasCycleCount(graphName: String, count: Long): Check = UnsupportedCheck
```